app/views/profile/users/edit.html.erb

```diff
@@ -24,7 +24,7 @@
           <%= check_box_tag :gluten_free, nil, @restrictions.include?('gluten free') %>
           <%= label_tag "gluten free" %><br>
           <%= check_box_tag :dairy_free, nil, @restrictions.include?('dairy free') %>
```

Comment:
We can refactor the `@restrictions.include?` methods here to use the new `has_favorited?` method you created.